components/ee/ws-scheduler/pkg/scaler/driver.go

```diff
@@ -151,6 +151,18 @@ func (wspd *WorkspaceManagerPrescaleDriver) Run() {
 		setpoint       int
 	)
 
+	canStartGhost := func() bool {
+		if status.Count.Ghost >= wspd.Config.MaxGhostWorkspaces {
+			log.WithField("limit", wspd.Config.MaxGhostWorkspaces).Warn("max number of ghost workspace reached")
+			return false
+		}
+		return true
+	}
+	registerStartingGhost := func(id string) {
+		startingGhosts[id] = wspd.time.Now()
+		status.Count.Ghost++
```

Comment:
I'm just realizing we re counting ghost twice now: IMO [L163](https://github.com/gitpod-io/gitpod/pull/3357/files#diff-4678ff698bf4d02fdd0f9423962d464d8f12c119a7f297987962f89945216769L163) should be removed.
@csweichel WDYS?

Reply:
where else do we count the ghost? Note that we never use `len(startingGhosts)`

Reply:
We do, in [L163](https://github.com/gitpod-io/gitpod/pull/3357/files#diff-4678ff698bf4d02fdd0f9423962d464d8f12c119a7f297987962f89945216769L163)

Reply:
`status.Count.Ghost += len(startingGhosts)`

Reply:
Thanks - now I've seen it :) 

Fixed